README.md

```diff
@@ -47,14 +47,15 @@ You can use `linuxkit run <name>` or `linuxkit run <name>.<format> to execute th
 This will use a suitable backend for your platform or you can choose one, for example VMWare.
 See `linuxkit run --help`.
 
-Additional, platform specific information is available for:
-- [macOS/HyperKit](docs/platform-hyperkit.md)
-- [qemu/kvm](docs/platform-qemu.md)
-- [Amazon Web Services](docs/platform-aws.md)
-- [Google Cloud](docs/platform-gcp.md)
-- [Microsoft Azure](docs/platform-azure.md)
-
-We'll add more detailed docs for other platforms in the future.
+Supported platforms are:
+
+| Local                                                 | Cloud                                       |
+|-------------------------------------------------------|---------------------------------------------|
+| [HyperKit (macOS)](docs/platform-hyperkit.md)         | [Amazon Web Services](docs/platform-aws.md) |
```

Comment:
prefer two lists to a table. They arent going to continue to have the same number of items in each, and harder to read.